service/application.go

```diff
@@ -324,3 +337,20 @@ func (app *Application) createMemoryBallast() ([]byte, uint64) {
 	}
 	return nil, 0
 }
+
+// updateActiveConfig shutdown the currently active configuration and loads and starts a new one.
+func (app *Application) updateActiveConfig(ctx context.Context) error {
```

Comment:
I am not totally sure the name reflects the intent. Should it be named "reloadConfig" instead? Why do we call it "active" config? Is there also an inactive config somewhere?

Reply:
Done, renamed to `updateService`